### PR TITLE
Package.json - Change repo name to pulsar-edit

### PR DESCRIPTION
**Pull request by LandarXT**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,7 +20,7 @@
   "license": "MIT",
   "repository": {
     "type": "git",
-    "url": "https://github.com/atom/docs.git"
+    "url": "https://github.com/pulsar-edit/docs.git"
```
Comment (confused-Techie) [inline package.json]:
```suggestion
    "url": "https://github.com/pulsar-edit/flight-manual.atom.io.git"
```

 I know this isn't what was done originally, but makes a lot more sense to me imo.

**Review by confused-Techie — CHANGES_REQUESTED** [review]
Added a change I think we can implement. Although not sure how others feel about it.

**Comment by LandarXT** [comment]
Seems to make sense to me

**Review by confused-Techie — APPROVED** [review]
Super rad, otherwise looks great.

**Comment by kaosine** [comment]
Honestly this one should just be archived in favor of the docs repo.....

**Comment by LandarXT** [comment]
I pull requested this one because it looked more complete compared to docs

**Comment by kaosine** [comment]
It's the same thing, only thing wrong is the system I moved the other to hates the file refs this one had. I've been trying to get around to getting it fixed but working around depression and everything else not giving me the ability to have the focus and energy I need....

**Review by ghost — APPROVED** [review]
Simple, no complaints. Easy.

**Pull request merged**